pydy/codegen/ode_function_generators.py

```diff
@@ -641,16 +641,26 @@ class CythonODEFunctionGenerator(ODEFunctionGenerator):
 
     def __init__(self, *args, **kwargs):
 
+        self._options = {'tmp_dir': None,
+                         'prefix': 'pydy_codegen',
+                         'cse': True,
+                         'verbose': False}
+        for k, v in self._options.items():
+            self._options[k] = kwargs.pop(k, v)
+
```

Comment:
Oh, no it is fine. Yeah, I could write it your way too. I'm not sure it really matters, as this is not a place that performance matters.